dev-util/flawfinder/flawfinder-2.0.7.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_6 )
+
+inherit distutils-r1
+
+DESCRIPTION="Examines C/C++ source code for security flaws"
+HOMEPAGE="https://www.dwheeler.com/flawfinder/"
+SRC_URI="https://www.dwheeler.com/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~sparc-solaris ~x86-solaris"
+IUSE="doc test"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="${PYTHON_DEPS}"
+RDEPEND="${DEPEND}"
+
+python_test() {
+	esetup.py test
+}
+
+python_install_all() {
+	local DOCS=( announcement ChangeLog README.md ${PN}.pdf )
+	doman ${PN}.1 || die 'doman failed.'
```

Comment:
I thought doman() calls die on it's own if it fails (maybe i'm wrong, haven't tested).